ultrasonic/src/main/res/layout/song_details.xml

```diff
@@ -55,7 +55,8 @@
             a:layout_gravity="left|center_vertical"
             a:layout_weight="1"
             a:ellipsize="middle"
-            a:paddingStart="4dip"
+            a:paddingStart="1dip"
+            a:paddingEnd="4dip"
```

Comment:
Details were too much to the left. 
Also added an end padding to not get to close to the duration.